src/main/java/me/marcellin/timechecker/api/v1/time/TimeController.java

```diff
@@ -16,13 +19,14 @@
     @Autowired
     private TimeService timeService;
 
-    @RequestMapping("/{queryCity}")
-    public TimeModel check(
+    @RequestMapping(value = "/{queryCity}", method = RequestMethod.GET,  produces = "application/json")
```

Comment:
spring uses that as a placeholder for data passed in through the url

example when you type `http://example.com/John Smith` in the browser, Spring will assign queryCity to John Smith